packages/components/src/public/QueryModel/GridPanel.tsx

```diff
@@ -748,11 +823,11 @@ export class GridPanel<T = {}> extends PureComponent<Props<T>, State> {
             showHeader,
             title,
         } = this.props;
-        const { showFilterModalFieldKey, actionValues } = this.state;
+        const { showFilterModalFieldKey, actionValues, errorMsg } = this.state;
         const { hasData, id, isLoading, isLoadingSelections, rowsError, selectionsError, messages, queryInfoError } =
             model;
         const hasGridError = queryInfoError !== undefined || rowsError !== undefined;
-        const hasError = hasGridError || selectionsError !== undefined;
+        const hasError = hasGridError || selectionsError !== undefined || errorMsg;
```

Comment:
should we be more specific with the `errorMsg` variable and call it `viewError` or something like that?

Reply:
Could do since that's the only other kind of error we currently capture, but if there are others in the future, the more generic name will suit, so I think I'll leave it as is since the logic isn't different for an error related to the view (at least not yet).